utils/testserver/index.js

```diff
@@ -66,6 +66,7 @@ class TestServer {
     this._wsServer.on('connection', this._onWebSocketConnection.bind(this));
     this._server.listen(port);
     this._dirPath = dirPath;
+    this.debugServer = require('debug')('pw:server');
```

Comment:
don't you want to have port here as well?

Reply:
also, can we make it private?

Reply:
I actually made it public so that it could be redirected. test harness can say `server.debugServer.log = data => output.push(data)` and that way we group messages by tests.